db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/CompilerContext.java

```diff
@@ -700,4 +701,8 @@
 	public void setAllowOverflowSensitiveNativeSparkExpressions(boolean newValue);
 
 	public boolean getAllowOverflowSensitiveNativeSparkExpressions();
+
+	public void setCurrentTimestampPrecision(int newValue);
+
+	public int getCurrentTimestampPrecision();
 }
```

Comment:
Timestamp precision  is a cached global database property, so it does not have to be a part of CompilerContext?

Reply:
It was my understanding that making it part of CompilerContext was _the_ way to access it in ActivationClassBuilder. Is there another way?

Reply:
I see. ActivationClassBuilder has no access to LanguageContext, that's why the property needs to be set on CompilerContext.